app/scenes/channel_info/channel_info_container.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import {getChannelStats} from 'service/actions/channels';
```

Comment:
It's totally fine to call it from `service` cause I don't see the need to validate if we already have this values in the store as we are probably better getting them again by this point